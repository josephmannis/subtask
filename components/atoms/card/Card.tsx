import React from 'react';
import styled from 'styled-components/native';

const Container = styled.View`
    border-radius: 14px;
    shadow-color: black;
    shadow-radius: 14px;
    shadow-opacity: .15;
    elevation: 12;
    background-color: white;
`

const Card: React.FC = props => {
    return (
        <Container>
            {props.children}
        </Container>
    )
}

export default Card;