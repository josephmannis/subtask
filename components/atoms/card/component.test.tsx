import React from 'react';
import renderer from 'react-test-renderer';
import Card from './Card';

test('Renders', () => {
    const tree = renderer.create(<Card/>).toJSON();
    expect(tree).toMatchSnapshot()
})