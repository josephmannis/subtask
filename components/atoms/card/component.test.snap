// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "white",
        "borderRadius": 14,
        "elevation": 12,
        "shadowColor": "black",
        "shadowOpacity": 0.15,
        "shadowRadius": 12,
      },
    ]
  }
/>
`;
